banyand/liaison/grpc/grpc.go

```diff
@@ -49,26 +49,35 @@ import (
 )
 
 var (
-	ErrSeriesEvents    = errors.New("no seriesEvent")
-	ErrShardEvents     = errors.New("no shardEvent")
-	ErrInvalidSeriesID = errors.New("invalid seriesID")
+	ErrSeriesEvents       = errors.New("no seriesEvent")
+	ErrShardEvents        = errors.New("no shardEvent")
+	ErrInvalidSeriesID    = errors.New("invalid seriesID")
+	ErrServerCert         = errors.New("invalid server cert file")
+	ErrServerKey          = errors.New("invalid server key file")
+	ErrServerHostOverride = errors.New("invalid serverHostOverride")
+	ErrNoAddr             = errors.New("no address")
 )
 
 type Server struct {
-	addr       string
-	log        *logger.Logger
-	ser        *grpclib.Server
-	pipeline   queue.Queue
-	repo       discovery.ServiceRepo
-	shardInfo  *shardInfo
-	seriesInfo *seriesInfo
+	addr               string
+	maxRecvMsgSize     int
+	TLSVal             bool
+	ServerHostOverride string
+	CertFile           string
```

Comment:
If we don't export them, switch them to lower case.

Reply:
I export them for  the `grpc_test`.

Reply:
You could follow the below piece to set those flags instead of exporting them:

```go
tcp.FlagSet().Parse([]string{"--tls=" + xxx})
```